cmp/RelativeTimestamp.js

```diff
@@ -39,13 +40,17 @@ const defaultOptions = {
 
 /**
  * A RelativeTimestamp component - It displays time relative to `now`
- *
- * @prop timestamp, Date
- * @prop options, Object, see the getRelativeTimestamp options
  */
-
 @hoistComponent()
 class RelativeTimestamp extends Component {
+
+    static propTypes = {
+        /** Date object that will be used as reference, can also be specified in milliseconds*/
+        timestamp: PT.oneOfType([PT.instanceOf(Date), PT.number]),
+        /** See getRelativeTimestamp} options */
+        options: PT.object
+    };
```

Comment:
I like the fancy typing on timestamp to express both acceptable types. Stray character in the options comment there.